trpcultivate_phenotypes/tests/src/Kernel/Display/DisplayValidationResultWindowTest.php

```diff
@@ -0,0 +1,406 @@
+                    '#header' => ['Header 1', 'Header 2'],
+                    '#attributes' => ['class' => ['tcp-raw-row']],
+                    '#rows' => [
+                      ['Row #1 - Value #1', 'Row #1 - Value #2'],
+                    ],
+                  ],
+                ],
+              ],
+            ],
+          ],
+        ],
+        ['tcp-validate-fail'],
+      ],
+      // #7: A failed validator: many rows in a table element.
+      [
+        'failed and many rows in a table element',
+        [
+          [
+            'title' => 'Validation Title Text - Fail',
+            'status' => 'fail',
+            'details' => [
+              '#type' => 'html_tag',
+              '#tag' => 'ul',
+              'lists' => [
+                [
+                  '#type' => 'html_tag',
+                  '#tag' => 'li',
+                  'table' => [
+                    '#type' => 'table',
+                    '#caption' => 'Validator Case Message',
+                    '#header' => ['Header 1', 'Header 2', 'Header 3'],
+                    '#attributes' => [],
+                    '#rows' => [
+                      ['Row #1 - Value #1', 'Row #1 - Value #2', 'Row #1 - Value #3'],
+                      ['Row #2 - Value #1', 'Row #2 - Value #2', 'Row #2 - Value #3'],
+                      ['Row #3 - Value #1', 'Row #3 - Value #2', 'Row #3 - Value #3'],
+                      ['Row #4 - Value #1', 'Row #4 - Value #2', 'Row #4 - Value #3'],
+                      ['Row #5 - Value #1', 'Row #5 - Value #2', 'Row #5 - Value #3'],
+                      ['Row #6 - Value #1', 'Row #6 - Value #2', 'Row #6 - Value #3'],
+                      ['Row #7 - Value #1', 'Row #7 - Value #2', 'Row #7 - Value #3'],
+                      ['Row #8 - Value #1', 'Row #8 - Value #2', 'Row #8 - Value #3'],
+                      ['Row #9 - Value #1', 'Row #9 - Value #2', 'Row #9 - Value #3'],
+                      ['Row #10 - Value #1', 'Row #10 - Value #2', 'Row #10 - Value #3'],
+                      ['Row #11 - Value #1', 'Row #11 - Value #2', 'Row #11 - Value #3'],
+                      ['Row #12 - Value #1', 'Row #12 - Value #2', 'Row #12 - Value #3'],
+                      ['Row #13 - Value #1', 'Row #13 - Value #2', 'Row #13 - Value #3'],
+                      ['Row #14 - Value #1', 'Row #14 - Value #2', 'Row #14 - Value #3'],
+                      ['Row #15 - Value #1', 'Row #15 - Value #2', 'Row #15 - Value #3'],
+                    ],
+                  ],
+                ],
+              ],
+            ],
+          ],
+        ],
+        ['tcp-validate-fail'],
+      ],
+      // #7: An image.
+      [
+        'an image',
+        [
+          [
+            'title' => 'Validation Title Text - Todo',
+            'status' => 'todo',
+            'details' => [
+              '#theme' => 'image_style',
+              '#style_name' => 'thumbnail',
+              '#uri' => __DIR__ . '/../Fixtures/png.png',
```

Comment:
The URI needs to be relative to the DrupalRoot in order to render properly but here you are providing the full directory path. Try using the DRUPAL_ROOT global variable instead of __DIR__.

Reply:
You've now changed it to `public://logo.svg` but since the test doesn't add anything here it makes this test brittle as it will fail if that file is not in a site running the tests.